packages/gestalt/src/index.js

```diff
@@ -13,6 +13,7 @@ import Collage from './Collage.js';
 import Collection from './Collection.js';
 import Column from './Column.js';
 import Container from './Container.js';
+import Datapoint from './Datapoint.js';
```

Comment:
@ponori just fyi, this is failing on Netlify / CI since the Linux file system is case sensitive. To fix the builds, we should change this (and other references) to `./DataPoint.js`

Mac OS file system is not case sensitive so that's why it works for you locally.